### PR TITLE
Update to support latest version of Intelli-J 2022.1

### LINKED ISSUE #53: Add support for 2022.1
After upgrading IntelliJ to version 2022.1, the plugin stopped working

image

### PR DESCRIPTION
**Pull request by bentemple**
## Summary
This updates the plugin to support the latest version of Intelli-J 2022.1.

This is mainly copied code from the following pull request in the parent repository [](https://github.com/JetBrains/intellij-platform-plugin-template): [IntelliJ Platform Plugin Template#240](https://github.com/JetBrains/intellij-platform-plugin-template/pull/240/files)

We could simply wait for the above [Pull Request](https://github.com/JetBrains/intellij-platform-plugin-template/pull/240/files) to land on main, and then rebase this repository on-top of it, but I wanted at the least to create a PR for other people to be able to build this project locally and continue using this very useful plugin.

Resolves #53

##Test-Plan
Ran `./gradlew build` locally
Build passed
Installed from local zip into latest version of Intelli-J 2022.1
Verified plugin functioned as expected

### CODE REVIEW

**Comment by pvgnd** [comment]
@WorkingDevel any chance you have a look on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,26 @@
 
 ## [Unreleased]
 ### Added
+### Changed
+
+## [2022.1.0]
```
Comment (WorkingDevel) [inline CHANGELOG.md:9]: This is generated by the changelog build pipeline step. So just leave this out and add your additions to ones under the `[Unreleased]` section.

**Review by WorkingDevel — CHANGES_REQUESTED** [review]
Thank you for work! Just the one thing with the changelog.

**Review by WorkingDevel — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bentemple) [reply CHANGELOG.md]: Thanks for the change / merge :)

**Pull request merged**